_ts/browse.ts

```diff
@@ -14,7 +14,7 @@ function start(keyword) {
     var widgets = document.querySelectorAll('.github-widget');
```

Comment:
While this is typescript (since ts is a superset of js), we didn't really take advantage of what ts offers -- static typing. Also consider using es6 scope let instead of global var.

Reply:
Yeah I have not yet completed the full conversion to ts, so far have just been able to convert the file extensions and debug the immediate fallout of doing so. I will speak with you at stand-up about the es6-es5 issues I have ran into as well